### DISCUSSION TITLE
How to disable asset manager modal on initial image drop ?

### DISCUSSION THREAD

**Discussion by anver** [GrapesJS/grapesjs#4954] [Q&A] [answered]
How to disable asset manager initially when dropping an image component on to the canvas ? The editor opens the asset manager when I drop the image component on to the canvas which is what in most case useful but I need to disable the asset manager opening automatically when dropping the image. I can later double click the placeholder image to open up the asset manager manually. How can i accomplish this ? Please help.

**Comment by artf** [comment, marked as answer]
If you're using the basic block plugin that is happening because the [block is configured](https://github.com/GrapesJS/blocks-basic/blob/ce61c45b9896d468bc8d4801b3edab8d282aa5a5/src/blocks.ts#L256) with the `activate` property. You can disable it by doing something like this:
```js
editor.Blocks.get('image').set('activate', false)
```
The double click behaviour is component related and independent from the block so it'll be preserved.
